### ISSUE TITLE
appcenter does not honor system proxy setting

### ISSUE THREAD

**Issue by tbachert** [elementary/appcenter#1239]
living in a network behind a http proxy prevents appcenter from contacting repositories, no matter that you configure the proxy in system settings (which i fine for all other apps).

possible workaround: add a proxy setting in the apt config, e.g. add following line to /etc/apt/apt.conf.d/70debconf

Acquire::http::Proxy "http://192.168.2.1:8123";

**Comment by cassidyjames** [comment]
I don't think AppCenter is doing any networking, aside from hitting the Houston server (using standard GLib networking) to display some apps on the home page. So I guess this issue would be that apt itself does not respect the proxy settings? Does apt work as expected from Terminal with your proxy enabled?

**Comment by tbachert** [comment]
exactly, apt in terminal also needs this manual proxy configuration, the problem may be located here

shouldn't the system config dialog not change the apt configuration?
